pymoskito/tools.py

```diff
@@ -264,3 +268,148 @@ def get_figure_size(scale):
     fig_height = fig_width * golden_ratio  # height in inches
     fig_size = [fig_width, fig_height]
     return fig_size
+
+def generate_binding(module_name, module_path):
+    src_path = os.path.join(os.path.dirname(module_path), "binding")
+    module_inc_path = os.path.join(src_path, module_name + ".h")
+    module_src_path = os.path.join(src_path, module_name + ".cpp")
+
+    pybind_path = os.path.join(os.path.dirname(__file__),
+                               os.pardir,
+                               "libs",
+                               "pybind11")
+    c_make_lists_path = os.path.join(src_path, 'CMakeLists.txt')
+
+    # check if folder exists
+    if not os.path.isdir(src_path):
+        logger.error("Dir binding not available in project folder '{}'"
+                     "".format(os.getcwd()))
+        return
+
+    if not os.path.exists(module_inc_path):
+        logger.error("Module '{}'.h could not found in binding folder"
+                     "".format(module_inc_path))
+        return
+
+    if not os.path.exists(module_inc_path):
+        logger.error("Module '{}'.h could not found in binding folder"
+                     "".format(module_src_path))
+        return
+
+    if not os.path.exists(c_make_lists_path):
+        logger.warning("No CMakeLists.txt found!")
+        logger.info("Generating new CMake config.")
+        create_cmake_lists(c_make_lists_path, pybind_path, module_name)
+
+    add_binding_config(c_make_lists_path, module_name)
+
+    # build
+    if os.name == 'nt':
```

Comment:
This will start a build process every time the wrapping class is loaded, which is done when the application is started.
Is this sufficient or would you like to retain the possibility to issue a new build when the object is actually instantiated?